tests/test_jx/test_deep_ops.py

```diff
@@ -820,9 +820,7 @@ def test_aggs_on_parent_and_child(self):
                     {"o": 1, "v": NULL, "s": 0},
                     {"o": 2, "v": "b", "s": 1},
                     {"o": 2, "v": "c", "s": 0},
-                    {"o": 2, "v": NULL, "s": 1},
-                    {"o": NULL, "v": "b", "s": 0},
-                    {"o": NULL, "v": "c", "s": 0}
+                    {"o": 2, "v": NULL, "s": 1}
```

Comment:
Looking into those changes again, it failed because I was trying to add those Null's in the wrong manner: [Query changes](https://github.com/mozilla/jx-sqlite/compare/master...rohit-rk:deep2#diff-0bb6ca9b5b562886608978394a262627R478). My approach was wrong. It made return lot of Null's out of nowhere.